Troika/Components/Button/Button+Style.swift

```diff
@@ -91,5 +91,12 @@ public extension Button {
             default: return .sardine
             }
         }
+
+        var margins: UIEdgeInsets {
+            switch self {
+            case .link: return UIEdgeInsets(top: .smallSpacing, left: 0, bottom: .smallSpacing, right: 0)
```

Comment:
Why does the link doesn’t have the same margins?

Reply:
Because then it will become difficult to align it with text other places because of its intrinsic margins. You would have to offset it with the side margin to align it. The margins are not visible anyways. Those were my thoughts at least.